DevDashboard/cloud/landing/obsidian-terminal/index.html

```
@@ -0,0 +1,821 @@
+          <span class="relative mb-6 inline-flex items-center gap-2 rounded-full border border-white/10 bg-white/[0.04] px-3 py-1 text-[10px] uppercase tracking-[0.2em] text-emerald-300/90">
+            <span class="live-dot h-1.5 w-1.5 rounded-full bg-emerald-400"></span> Ship with confidence
+          </span>
+          <h2 class="relative mx-auto max-w-2xl font-display text-4xl font-semibold leading-tight tracking-[-0.02em] text-zinc-50 md:text-5xl">
+            Put your dev machine in your pocket.
+          </h2>
+          <p class="relative mx-auto mt-5 max-w-xl text-lg text-zinc-400">
+            Your machine. Your keys. We can't see your data — and you can prove it.
+          </p>
+          <form class="relative mx-auto mt-10 flex max-w-md flex-col items-center gap-3 sm:flex-row" onsubmit="return false;">
+            <div class="w-full rounded-full border border-white/10 bg-white/[0.04] p-1.5 sm:flex-1">
+              <input type="email" required placeholder="you@yourmachine.dev" class="w-full rounded-full bg-transparent px-4 py-2.5 text-sm text-zinc-100 placeholder:text-zinc-600 focus:outline-none" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)